lib/system/environ/validate.go

```diff
@@ -46,9 +46,9 @@ func ValidateInstall(env *localenv.LocalEnvironment) func() error {
 			log.WithError(err).Warn("Failed to validate state directory requirements.")
 		}
 		// FIXME: validate enough of directory contents to make an educated decision automatically
-		// if err := validateDirectoryEmpty(stateDir); err != nil {
```

Comment:
I think it's better on though this has a downside of not allowing the user to restart `gravity install` after it has created the state directory but before it could start an operation. This will require manual state directory removal with `./gravity leave --force` (which not that bad and there's a corresponding message about it).